### ISSUE TITLE
Code exists abruptly while running pretrained checkpoints

### ISSUE THREAD

**Issue by skrya** [dcharatan/pixelsplat#28]
I have used the following command to run inference. "python3 -m src.main +experiment=acid mode=test dataset/view_sampler=evaluation checkpointing.load=./acid.ckpt". I have checked the path to the dataset and it is correct. 

The code stops after issuing a warning (not error) as below. 

[2024-02-09 17:55:47,950][py.warnings][WARNING]  Your `IterableDataset` has `__len__` defined. In combination with multi-process data loading (when num_workers > 1), `__len__` could be inaccurate if each worker is not configured independently to avoid having duplicate data.

I am not sure why the code just exits without giving a trace. Let me know if there is something missing in getting it to work.

**Comment by dcharatan** [comment]
Have you confirmed whether it's printing any frames to the `outputs/test` folder? If you're using a small data subset, it's possible that it finishes running without printing anything.

**Comment by skrya** [comment]
Thanks for the quick response. It does not print any frames. Output/acid folder just has benchmark.json and peakmemory.json. 
Should I try the larger dataset test folder just in case. Can you share it if you have it available somewhere to ensure I can reproduce?

**Comment by dcharatan** [comment]
I would recommend trying out the Real Estate 10k test set as well, since it's possible that the evaluation index used for ACID in the paper doesn't have overlap with the ACID data subset. I can also share a link to the full datasets we used to train and evaluate the model if you email me at charatan@mit.edu (we can't post the link publicly, unfortunately).

**Comment by dcharatan** [comment]
For anyone who stumbles across this: to evaluate on ACID, you have to use the correct evalution index.

```
python3 -m src.main +experiment=acid mode=test dataset/view_sampler=evaluation dataset.view_sampler.index_path=assets/evaluation_index_acid.json checkpointing.load=checkpoints/acid.ckpt
```

For more details, see #38.
